### DISCUSSION TITLE
add custom notes area

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2983] [Ideas] [answered]
The Overview web page is currently used by the SO team to post changes as updates occur. It would really be helpful if there were a div or some other code box that pulls from a static file that doesn't get overwritten during updates so Admin's can add their own links and notes for easy reference.  There's diagrams and IP documents that I reference often when hunting and it would be most helpful to have those links handy directly in the SO interface.  

Or perhaps a better method would be to have a 'Notes' section accessible from the 'Administration' menu tree.  It could be as simple as an editable json file where we can code anything we want.  But the requirement is that it doesn't get overwritten upon updating.

Currently I'm editing `/opt/so/saltstack/default/salt/soc/files/soc/changes.json` and then waiting for salt to update and it posts my edits to the webpage. I suspect it will be overwritten during next soup.

**Comment by TOoSmOotH** [comment, marked as answer]
All you need to do is copy it to /opt/so/saltstack/local/soc/files/soc/changes.json and it will not get overwritten when the update happens. We plan on moving the change log to an icon in the top right so that real estate can be customized.
